source/includes/qe/make-data-key.js

```diff
@@ -0,0 +1,79 @@
+const { MongoClient, Binary } = require("mongodb");
+const { ClientEncryption } = require("mongodb-client-encryption");
+
+const eDB = "encryption";
+const eKV = "__keyVault";
+const keyVaultNamespace = `${eDB}.${eKV}`;
+const secretDB = "medicalRecords";
+const secretCollection = "patients";
+
+const fs = require("fs");
+const crypto = require("crypto");
+try {
+  fs.writeFileSync("master-key.txt", crypto.randomBytes(96));
+} catch (err) {
+  console.error(err);
+}
+
+const provider = "local";
+const path = "./master-key.txt";
+// WARNING: Do not use a local key file in a production application
+const localMasterKey = fs.readFileSync(path);
+const kmsProviders = {
+  local: {
+    key: localMasterKey,
+  },
+};
+
+async function run() {
+  const uri = "<Your Connection String>";
+  const keyVaultClient = new MongoClient(uri);
+  await keyVaultClient.connect();
+  await keyVaultColl.createIndex(
+    { keyAltNames: 1 },
+    {
+      unique: true,
+      partialFilterExpression: { keyAltNames: { $exists: true } },
+    }
+  );
+  const clientEnc = new ClientEncryption(keyVaultClient, {
+    keyVaultNamespace: keyVaultNamespace,
+    kmsProviders: kmsProviders,
+  });
+  const dek1 = await clientEnc.createDataKey(provider, {
```

Comment:
For most programming languages, the style guides recommend staying away from numbers in variable names.